docusaurus.config.js

```diff
@@ -1,5 +1,7 @@
 require("dotenv").config();
 
+const isUpdate = process.env.UPDATE_SALEOR;
```

Comment:
What does it mean `isUpdate` and `UPDATE_SALEOR`?
Is it documented anywhere?

Reply:
It's used in the `Releasing Canary to 3.x` section in README and it allows properly re-create `3.x` based on `Canary`